sdk/analytics/src/main/java/com/klaviyo/analytics/linking/DeepLinking.kt

```diff
@@ -16,7 +16,9 @@ import com.klaviyo.core.lifecycle.LifecycleMonitor.Companion.ACTIVITY_TRANSITION
  * Callback type for handling a deep link. When registered, this callback is invoked with any
  * deep links originating from Klaviyo services, instead of broadcasting an [Intent].
  */
-typealias DeepLinkHandler = (uri: Uri) -> Unit
+fun interface DeepLinkHandler {
+    operator fun invoke(uri: Uri)
+}
```

Comment:
I'm not 🤓 enough about shrinkers to fully explain it, but using an interface instead of typealias seems to preserve the type or something, so that even in a release build you can safely use `Registry.register<DeepLinkHandler>()` and `Registry.get<DeepLinkHandler>()`